Documentation/faq.md

```diff
@@ -38,6 +62,12 @@ It is recommended to have an odd number of members in a cluster. An odd-size clu
 
 Adding a member to bring the size of cluster up to an even number doesn't buy additional fault tolerance. Likewise, during a network partition, an odd number of members guarantees that there will always be a majority partition that can continue to operate and be the source of truth when the partition ends.
 
+#### Can I deploy ecd cross-region or cross data center?
+
+Yes, you can. Deploying etcd across regions add resilience to etcd by putting members into different failure domain. The cost is higher request latency. If a client in data center A talks to an etcd member in data center B, it might see higher latency. Also internally etcd replicates data across all its peers, so the latency increases.
```

Comment:
Deploying etcd across regions improves etcd's fault tolerance since members are in separate failure domains. The cost is higher consensus request latency from crossing data center boundaries. Since etcd relies on a member quorum for consensus, the latency from crossing data centers will be somewhat pronounced because at least a majority of cluster members must respond to consensus requests. Additionally, cluster data must be replicated across all peers, so there will be bandwidth cost as well.